### PR TITLE
Endpoint name mandatory

### PR DESCRIPTION
**Pull request by SimonCropp**
also
 
 * fix copyright
 * fix nuspecs
 * remove some redundant code
 * fix name of test project
 * update fody

replaces https://github.com/Particular/NServiceBus.Host.AzureCloudService/pull/23

### CODE REVIEW

**Comment by SimonCropp** [comment]
@Particular/azure-maintainers ready for a review/merge

File: packaging/nuget/nservicebus.hosting.azure.hostprocess.nuspec [hunk 1]
```diff
@@ -14,13 +14,13 @@
...
-      <dependency id="CommonServiceLocator" version="[1.0.0.0,1.3]" />
-      <dependency id="Microsoft.Data.OData" version="[5.6.0.0, 6.0.0.0)" />
-      <dependency id="Microsoft.WindowsAzure.ConfigurationManager" version="[2.0.1.0, 3.0.0.0)" />
-      <dependency id="NServiceBus" version="[6.0.0.0-unstable, 7.0.0.0)" />
-      <dependency id="System.Spatial" version="[5.6.0.0, 6.0.0.0)" />
-      <dependency id="WindowsAzure.Storage" version="[3.1.0.1, 5.0.0.0)" />
-      <dependency id="NServiceBus.Hosting.Azure" version="[$version$, 7.0.0.0)" />
+      <dependency id="CommonServiceLocator" version="[1.0.0,1.3]" />
+      <dependency id="Microsoft.Data.OData" version="[5.6.4, 6.0.0)" />
+      <dependency id="Microsoft.WindowsAzure.ConfigurationManager" version="[2.0.3, 3.0.0)" />
```
Comment (melkio) [inline packaging/nuget/nservicebus.hosting.azure.hostprocess.nuspec]: Why not the latest version available (3.2.1)?

File: packaging/nuget/nservicebus.hosting.azure.nuspec [hunk 2]
```diff
@@ -4,25 +4,25 @@
...
+    <authors>$authors$</authors>
+    <owners>$owners$</owners>
+    <licenseUrl>$licenseUrl$</licenseUrl>
+    <projectUrl>$projectUrl$</projectUrl>
+    <iconUrl>$iconUrl$</iconUrl>
+    <requireLicenseAcceptance>$requireLicenseAcceptance$</requireLicenseAcceptance>
     <description>The Azure Cloudservices Host for NServicebus</description>
-    <copyright>Copyright 2010-2014 NServiceBus. All rights reserved</copyright>
+    <copyright>$copyright$</copyright>
     <tags>nservicebus servicebus msmq cqrs publish subscribe</tags>
```
Comment (SeanFeldman) [inline packaging/nuget/nservicebus.hosting.azure.nuspec]: I recall conversation about $tags variable. Should we replace hardcoded valued with a variable?

File: packaging/nuget/nservicebus.hosting.azure.hostprocess.nuspec (hunk 1 above)
Comment (SimonCropp) [inline packaging/nuget/nservicebus.hosting.azure.hostprocess.nuspec]: that ref is actually not required. i will remove it

File: packaging/nuget/nservicebus.hosting.azure.nuspec (hunk 2 above)
Comment (SimonCropp) [inline packaging/nuget/nservicebus.hosting.azure.nuspec]: @SeanFeldman i dont know how to do that

Comment (SeanFeldman) [inline packaging/nuget/nservicebus.hosting.azure.nuspec]: @danielmarbach I think you've mentioned something in a different repo about $tags. Could you let us know how to do it? Thanks.

**Comment by SimonCropp** [comment]
Green build pending... but i think this is good to go. tags can be fixed ltr

File: packaging/nuget/nservicebus.hosting.azure.nuspec (hunk 2 above)
Comment (danielmarbach) [inline packaging/nuget/nservicebus.hosting.azure.nuspec]: @SeanFeldman https://github.com/Particular/NuGetPackager/blob/master/ReadMe.md

**Pull request merged**